src/Benchmarks.ClientJob/ClientJob.cs

```diff
@@ -60,7 +63,7 @@ public ClientJob(ClientJob clientJob)
         public ClientState State { get; set; }
 
         public double RequestsPerSecond { get; set; }
-        public int Requests { get; set; }
+        public int TotalRequestsForJob { get; set; }
```

Comment:
Breaking change?

It's fine adding new properties and having the Driver and services using these properties, but this requires everyone to get the new Driver too, if it can be prevented then we should.

Reply:
If we know that drivers will require a breaking change, then we can do as many as we want though.

Reply:
I'll revert this